libraries/AP_HAL_ChibiOS/hwdef/VUAV-V7pro/README.md

```diff
@@ -0,0 +1,607 @@
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>RX (IN)</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>SCL I2C4</td>
+   <td>+3.3V (pullups)</td>
+   </tr>
+   <tr>
+   <td>5</td>
+   <td>SDA I2C4</td>
+   <td>+3.3V (pullups)</td>
+   </tr>
+   <tr>
+   <td>6</td>
+   <td>SafetyButton</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>7</td>
+   <td>SafetyLED</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>8</td>
+   <td>-</td>
+   <td>-</td>
+   </tr>
+   <tr>
+   <td>9</td>
+   <td>Buzzer</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>10</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+### GPS2/I2C3 port
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>TX (OUT)</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>RX (IN)</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>SCL I2C3</td>
+   <td>+3.3V (pullups)</td>
+   </tr>
+   <tr>
+   <td>5</td>
+   <td>SDA I2C3</td>
+   <td>+3.3V (pullups)</td>
+   </tr>
+   <tr>
+   <td>6</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+### CAN1,CAN2 port
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>  
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>CAN_H</td>
+   <td>+24V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>CAN_L</td>
+   <td>+24V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+### I2C1,I2C2 port
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>SCL</td>
+   <td>+3.3 (pullups)</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>SDA</td>
+   <td>+3.3 (pullups)</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+### USB Ex
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin </th>
+   <th>Signal </th>
+   <th>Volt </th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC IN</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>D_minus</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>D_plus</td>
+   <td>+3.3V</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td>GND</td>
+   </tr>
+   </tbody>
+   </table>
+
+### RSSI
+
+   <table border="1" class="docutils">
+   <tbody>
+   <tr>
+   <th>Pin</th>
+   <th>Signal</th>
+   <th>Volt</th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>VCC</td>
+   <td>+5V</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>RSSI</td>
+   <td>up to +3.3V</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>SBUS_OUT</td>

```

Comment:
what uart pin is this, and no uarts are setup to provide SBUS out protocol in the hwdef

Reply:
This SBUS, DSM as the same port，Can be used as a universal serial port output

Reply:
It should be named UART8 TX unless you default UART8 protocol to "15" for sbus servo output....and if you really have UART8 RX tied the the same pin as RCIN, we should note that in the UART section....in which case you could set UART8 protocol to 23 (RCIN) and use its TX pin (currently mislabeled as SBUS_OUT) anf the RCIN pin for CRSF/ELRS,etc.